content/de/cloud-computing.md

```diff
@@ -5,17 +5,20 @@ category: Konzept
 ---
 
 ## Was es ist
+
 Das Konzept des Cloud Computing ist die Bereitstellung von Systemressourcen wie CPU, Netzwerk und Speicher auf Abruf und über das Internet.
 Cloud Computing ermöglicht den Zugriff und die Nutzung von Systemressourcen aus beliebigen Standorten.
 Diese Systemressourcen werden von Cloud-Anbietern wie AWS, GCP, Azure, Digitial Ocean und weiteren Anbietern an den Endverbraucher zur Verfügung gestellt.
```

Comment:
"an den Endverbraucher zu Verfügung gestellt." Ist das grammatikalisch korrekt?

"dem Endverbraucher zur Verfügung gestellt." klingt besser.

Reply:
stimmt. Es ging in diesem PR allerdings weniger um den Inhalt, als viel mehr darum semantic line breaks und leerzeilen einzubauen.